### PR TITLE
8191758: Match WebKit's font weight rendering with JavaFX

### PR DESCRIPTION
**Pull request by bhaweshkc**
As per JavaFx 700 font weight is considered to be bold but webkit is using 600 font weight for text to become bold. to fix issue, use boldWeightValue() function which uses 700 font weight rather than isFontWeightBold() which compare against 600 font weight.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8191758](https://bugs.openjdk.java.net/browse/JDK-8191758): Match WebKit's font weight rendering with JavaFX

### Reviewers
 * Kevin Rushforth ([kcr](@kevinrushforth) - **Reviewer**)
 * Arun Joseph ([ajoseph](@arun-Joseph) - Committer)

### Download
`$ git fetch https://git.openjdk.java.net/jfx pull/180/head:pull/180`
`$ git checkout pull/180`

### CODE REVIEW

**Comment by kevinrushforth** [comment]
Can you add a unit test to go along with this fix?

**Comment by prrace** [comment]
Per the opentype spec, 700 is bold. 600 is semi-bold
https://docs.microsoft.com/en-us/typography/opentype/spec/os2#usweightclass

CSS agrees : https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight

So are you saying webkit has been using bold at a lower weight than these specs suggest ?
I see the logic all comes from Source/WebCore/platform/graphics/FontSelectionAlgorithm.h

I suppose the existing code thinks that if we have reached what that file calls the bold threshold of 600 then we should use bold. It isn't necessarily "wrong" but I think I agree that it is more important to be consistent with the rest of Java FX ... which I believe is the point of this change ?

**Comment by bhaweshkc** [comment]
@prrace 
yes, webkit is set to use 600 weight where javafx consider 700 weight for it to be consider bold.
https://docs.oracle.com/javafx/2/api/javafx/scene/text/FontWeight.html

File: modules/javafx.web/src/test/java/test/javafx/scene/web/WebViewTest.java [hunk 1]
```diff
@@ -89,4 +90,28 @@ private void setZoom(final WebView view, final float zoom) throws Exception {
...
+            "<body>" +
+            "   <div style=\"font: 19px system-ui\">" +
+            "       <div style=\"font-style: italic;\">" +
+            "           <span id=\"six\" style=\"font-weight: 600;\">Hello, World</span>" +
+            "           <span id=\"nine\" style=\"font-weight: 900;\">Hello, World</span>" +
+            "       </div>" +
+            "   </div>" +
+            "</body> </html>"
+            );
+        submit(()->{
```
Comment (kevinrushforth) [inline modules/javafx.web/src/test/java/test/javafx/scene/web/WebViewTest.java:111]: Minor: there should be a space before and after the `->`

**Pull request closed without merging**